### ISSUE TITLE
Tray icon not visible on GNOME

### ISSUE THREAD

**Issue by yochananmarqos** [boomshop/pulseffectstray#2]
All I get is the following output with no tray icon: 

```
/usr/sbin/pulseeffectstray:387: DeprecationWarning: Gtk.StatusIcon.set_visible is deprecated
  self.icon.set_visible(True)
/usr/sbin/pulseeffectstray:470: DeprecationWarning: Gtk.StatusIcon.set_from_pixbuf is deprecated
  self.icon.set_from_pixbuf(self.image_active)
```

Manjaro GNOME
AppIndicator Support extension

**Comment by boomshop** [comment]
The warnings are because GTK will drop StatusIcon in the near future, which is the reason for PulseEffects will not build in one - and for setting up this small side project.

AFAIK (I don't have any clue about current Gnome desktop) one has to enable the status icon somehow by configuration/application. Nothing fancy to this application, just a standard status icon.

https://wiki.gnome.org/Initiatives/StatusIconMigration/Guidelines
https://www.reddit.com/r/linux/comments/6x8y7l/status_icons_and_gnome/

**Comment by yochananmarqos** [comment]
Right, as I said I'm using the [AppIndicator Support](https://extensions.gnome.org/extension/615/appindicator-support/) extension. All my other tray icons work fine.

**Comment by boomshop** [comment]
The amount of non-working icons (see issues) might indicate a problem in the extension itself.
